lib/Common/Exceptions/Throw.h

```diff
@@ -208,6 +208,10 @@ namespace Js {
         {       \
             hr = GetRuntimeErrorFunc(Js::RecyclableObject::FromVar(errorObject), nullptr);   \
         }   \
+        else if (errorObject != nullptr) \
```

Comment:
Just curious of the bug's effect... where do I see an OOM if my script "throw 123"? Tried Edge, F12, ch -- none shows OOM?